src/components/PostsTable.js

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable */
+import React from 'react';
+import styled from 'styled-components';
+import dayjs from 'dayjs';
+
+function PostsTable({ posts }) {
+  const ui =
+    posts &&
+    posts.map((post) => (
```

Comment:
Is there a reason you don't render this directly into the `tbody` instead of assigning it to the `ui` variable?

Reply:
Btw you can also use the `?` to get rid of the `&&` check. So instead of `posts && posts.map` you can write `posts?.map`

Reply:
purely for aesthetic reasons since render function was getting long. should i not do that?
thanks for the tip on '?'

Reply:
however when i make the change my code breaks with the following error
Cannot read property 'value' of null

Reply:
Hmm interesting, no idea why it doesn't work tbh. The variable is maybe also a personal preference from my side. Doesn't change functionality in the end. Just fine it easier to read. So you may as well keep it the way it is :wink: